frontend/common/src/helpers/router.tsx

```diff
@@ -102,6 +102,7 @@ export const useRouter = (
   routes: Routes<RouterResult>,
   missingRouteComponent: ReactElement,
   notAuthorizedComponent: ReactElement,
+  welcomeRoute?: string,
```

Comment:
@tristan-orourke I think this was what you were referring to, passing the route to the router hook?

Reply:
Yep, that's pretty much what I was thinking.